cameo/strain_design/heuristic/evolutionary/processing.py

```diff
@@ -95,17 +95,16 @@ def process_gene_knockout_solution(model, solution, simulation_method, simulatio
         A list with: reactions, genes, size, fva_min, fva_max, target flux, biomass flux, yield, fitness
     """
 
-    with TimeMachine() as tm:
+    with model:
```

Comment:
The other method bellow (process_reaction_swap_solution) needs to be replaced too.